rmw_fastrtps_shared_cpp/include/rmw_fastrtps_shared_cpp/custom_event_info.hpp

```diff
@@ -56,6 +58,8 @@ class EventListenerInterface
     return event_guard[event_type];
   }
 
+  virtual void update_inconsistent_topic(uint32_t total_count, uint32_t total_count_change) = 0;
```

Comment:
I can't comment on the appropriate line, but I think `take_event`'s docstring is outdated

Reply:
Oh, never mind,  I actually fixed it in a0919117f9db938c217dc1034165b035bcd4e0db .